doc/guides/contributing/issues.md

```diff
@@ -90,10 +90,10 @@ including whether the behavior being seen is a bug or a feature. This discussion
 is part of the process and should be kept focused, helpful, and professional.
 
 Short, clipped responses that provide neither additional context nor supporting
-detail are not helpful or professional. To many, such responses are simply
+details are not helpful or professional to many. Such responses are simply
```

Comment:
```suggestion
details are not helpful or professional. To many, Such responses are simply
```

I personally prefer the original sentence break here, but `details` definitely needs to be pluralized